Source/Core/DolphinQt/Config/Graphics/GeneralWidget.cpp

```diff
@@ -251,17 +251,23 @@ void GeneralWidget::AddDescriptions()
                  "a separate render window.<br><br><dolphin_emphasis>If unsure, leave "
                  "this unchecked.</dolphin_emphasis>");
   static const char TR_ASPECT_RATIO_DESCRIPTION[] = QT_TR_NOOP(
-      "Selects which aspect ratio to use when drawing on the render window.<br>"
-      "Each game can have a slightly different native aspect ratio.<br>They can vary by "
-      "scene and settings and rarely ever exactly match 4:3 or 16:9."
-      "<br><br><b>Auto</b>: Uses the native aspect ratio"
-      "<br><br><b>Force 16:9</b>: Mimics an analog TV with a widescreen aspect ratio."
-      "<br><br><b>Force 4:3</b>: Mimics a standard 4:3 analog TV."
-      "<br><br><b>Stretch to Window</b>: Stretches the picture to the window size."
-      "<br><br><b>Custom</b>: Forces the specified aspect ratio."
-      "<br>This is mostly intended to be used with aspect ratio cheats/mods."
-      "<br><br><b>Custom (Stretch)</b>: Similar to `Custom` but not relative to the "
-      "title's native aspect ratio.<br>This is not meant to be used under normal circumstances."
+      "Selects which aspect ratio to use for displaying the game."
+      "<br><br>The aspect ratio of the picture sent out by the original consoles varied depending "
+      "on the game and rarely exactly matched 4:3 or 16:9. Some of the picture would be cut off by "
+      "the edges of the TV, or the picture wouldn't fill the TV entirely. By default, Dolphin "
+      "shows the whole picture without distorting the proportions, which means it's normal for the "
+      "picture to not entirely fill your display."
+      "<br><br><b>Auto</b>: Mimics a TV with either a 4:3 or 16:9 aspect ratio, depending on which "
+      "type of TV the game seems to be targeting."
+      "<br><br><b>Force 16:9</b>: Mimics a TV with a 16:9 (widescreen) aspect ratio."
+      "<br><br><b>Force 4:3</b>: Mimics a TV with a 4:3 aspect ratio."
+      "<br><br><b>Stretch to Window</b>: Stretches the picture to the window size. "
+      "This will usually make proportions look distorted."
+      "<br><br><b>Custom</b>: Mimics a TV with the specified aspect ratio. "
+      "This is mostly intended to be used with aspect ratio cheats/mods."
+      "<br><br><b>Custom (Stretch)</b>: Similar to `Custom`, but stretches the picture as if to "
```

Comment:
I'm not very happy with "stretches the picture as if to complete fill the TV".

How about...

> Similar to 'Custom', but stretches the image to the specified aspect ratio. This will usually distort the image, and should not be used under normal circumstances.

Reply:
Do you have a preference for "picture" vs "image"? Either would be fine to me, but I think it looks better if we consistently use the same term throughout the description.

Reply:
I personally prefer image.

It's a very subtle english language thing, probably even regional, but to me, "picture" is the absolute final result after everything including display (literally photons into eyeballs level), whereas "image" is more generally referring to the pixels being made and sent to the display. For example, "that television has a nice picture" is referring to the televisions ability to emit the signal that was sent to it, while "*points at television* that's a nice image" is referring to what the signal being sent to the television. "that television has a nice image" is not a thing in american english, as far as I know.

I'm not sure the origins of that, I believe it's from television nomenclature? Language is weird. ┐(´-｀)┌

Reply:
I applied your suggestion, except I changed "distort the image" to "distort the image's proportions" to make it clearer what kind of distortion we're talking about. "Distort the image" on its own feels to me like to could be referring to anything from signal quality degradation to the image being cut off or sheared.

Also tweaked the rest of the description to keep the terminology consistent.